src/api/todos.ts

```diff
@@ -0,0 +1,20 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+// https://mate.academy/students-api/todos?userId=6345
+
+export const createTodo = (title: string, userId: number): Promise<Todo> => {
+  return client.post('/todos', {
+    userId,
+    title,
+    completed: false,
+  });
+};
```

Comment:
let us just whole object to function

Reply:
Don't really get what you mean. Could you, please, explain?